include/gc/gc.h

```diff
@@ -605,6 +605,20 @@ GC_API int GC_CALL GC_posix_memalign(void ** /* memptr */, size_t /* align */,
 /* GC_free(0) is a no-op, as required by ANSI C for free.               */
 GC_API void GC_CALL GC_free(void *);
 
+# if defined(VALGRIND_TRACKING)
```

Comment:
By convention, ifdef is not use in public header, where possible (except for ifdef GC_THREADS), please remove it. See the next comment.

Reply:
Fixed.